src/test/java/com/google/sps/CollectVideosTest.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.when;
+import org.mockito.Mock;
+import java.util.*;
+
+@RunWith(JUnit4.class)
+public final class CollectVideosTest {
+
+    @Test
+    public void MockMethodsOfDoGet(){
+        HttpServletRequest RequestMock = Mockito.mock(HttpServletRequest.class);
+        Room RoomMock = Mockito.mock(Room.class);
+        // When the doGet asks for the RoomId, return the hard-coded String 456
+        when(RequestMock.getParameter(ServletUtil.INPUTTED_ID_TAG)).thenReturn("456");
```

Comment:
move string and room id integer as global variables. This way multiple tests can re-use this id.

Reply:
rewrote the collectVideosTest file